### PR TITLE
feat(DWS): add a new resource to manage event subscription

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
Running tool: /usr/local/go/bin/go test -timeout 9000000000s -run ^TestAccDwsEventSubs_basic$ github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws -v

=== RUN   TestAccDwsEventSubs_basic
=== PAUSE TestAccDwsEventSubs_basic
=== CONT  TestAccDwsEventSubs_basic
--- PASS: TestAccDwsEventSubs_basic (30.94s)
PASS
ok  	github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dws	31.048s

```

### CODE REVIEW

File: huaweicloud/services/acceptance/dws/resource_huaweicloud_dws_event_subscription_test.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+	return fmt.Sprintf(`
+resource "huaweicloud_smn_topic" "test" {
+  name         = "%s"
+  display_name = "The display name of topic"
+}
+
+resource "huaweicloud_dws_event_subscription" "test" {
+  name   = "%s"
+  enable = 1
+  notification_target      = huaweicloud_smn_topic.test.id
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/dws/resource_huaweicloud_dws_event_subscription_test.go]:
```suggestion
  name                     = "%s"
  enable                   = 1
  notification_target      = huaweicloud_smn_topic.test.id
```

File: huaweicloud/services/acceptance/dws/resource_huaweicloud_dws_event_subscription_test.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+	return fmt.Sprintf(`
+resource "huaweicloud_smn_topic" "test" {
+  name         = "%s"
+  display_name = "The display name of topic"
+}
+
+resource "huaweicloud_dws_event_subscription" "test" {
+  name   = "%s"
+  enable = 0
+  notification_target      = huaweicloud_smn_topic.test.id
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/dws/resource_huaweicloud_dws_event_subscription_test.go]: ditto.

File: huaweicloud/services/acceptance/dws/resource_huaweicloud_dws_event_subscription_test.go (hunk 1 above)
Comment (hmbig2) [reply huaweicloud/services/acceptance/dws/resource_huaweicloud_dws_event_subscription_test.go]: done

**Pull request merged**